### ISSUE TITLE
Console loading state

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#4365]
**Description:**

We do not currently show when the console is loading. This happens at the beginning while the logs are being fetched and then streaming in.

**Comment by jonbell-lot23** [comment]
[Figma link](https://www.figma.com/file/SSr1ljyzF0lCXTL7BX1Whh/?node-id=9166%3A95466)

![image](https://user-images.githubusercontent.com/9154902/143505685-2e7c0adc-d06e-48b2-a2d3-1f8ad9b0bbf8.png)

In other panes, we're using a spinner because of the rounded nature of the panel. But in this case, we can stick with our progress bar and I think it looks better here. (Open to the spinner too, don't feel super strongly about it)

**Comment by jaril** [comment]
Progress bar sounds good here. Loading happens _a lot_ in response to a user action (adding a print statement, evaluating something in the console, adding event breakpoints, etc). A traditional spinner would block out the entire thing.

**Comment by jasonLaster** [comment]
Good call. For the first use case we can add redux field that is true while we are waiting for the fetch console logs promise to resolve. Down the line it could also be true when we are waiting for print statement logs or event listeners

**Comment by jonbell-lot23** [comment]
Ah, great point Jaril
